### PR TITLE
thriftgo 0.2.0  (new formula) 

### PR DESCRIPTION
**Pull request by Rainshaw**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by dawidd6 — APPROVED** [review]

File: Formula/thriftgo.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class Thriftgo < Formula
+  desc "Implementation of thrift compiler in go language with plugin mechanism"
+  homepage "https://github.com/cloudwego/thriftgo"
+  url "https://github.com/cloudwego/thriftgo/archive/refs/tags/v0.2.9.tar.gz"
```
Comment (chenrui333) [inline Formula/thriftgo.rb:4]: this did not get  marked as latest release, https://github.com/cloudwego/thriftgo/releases/tag/v0.2.9

Comment (Rainshaw) [reply Formula/thriftgo.rb:4]: ok, I created a issue for this https://github.com/cloudwego/thriftgo/issues/91

Comment (Rainshaw) [reply Formula/thriftgo.rb]: @chenrui333 hi, I have changed the tag to v0.2.0 as the maintainer of cloudwego/thriftgo wants to make a release for v0.3.0

Comment (chenrui333) [reply Formula/thriftgo.rb]: sounds good

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@Rainshaw, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**